website/components/Navbar.vue

```diff
@@ -37,6 +42,11 @@ export default {
       },
     };
   },
+  computed: {
+    routeURL() {
+      return this.$route.name !== "index";
+    },
```

Comment:
We may want to rename this to "showStateNav" or something similar so it's obvious that this is a boolean. I was just working on this, and then I saw you made it!